ui/client/datasets/DataTransformation/DataTransformation.js

```diff
@@ -181,6 +189,60 @@ const DataTransformation = withStyles(() => ({
   // eslint-disable-next-line no-unused-vars
   const [failedJobs, setFailedJobs] = useState([]);
 
+  // toggles when the RunTransformations overlay closes
+  const [allTransformationsComplete, setAllTransformationsComplete] = useState(false);
+  // toggles when the RunTransformation component unmounts due to errors
+  const [transformationProcessingError, setTransformationProcessingError] = useState(false);
+
+  // Handles the next step process after the transformations are complete in <RunTransformation>
+  useEffect(() => {
```

Comment:
I don't follow this useEffect section, as there is a different `handleNextStep` that we need to call. Not sure what `handleNext` is in this context. 😕